### ISSUE TITLE
Is there a method similar to "gl.getActiveXXX" in ShaderModule?

### ISSUE THREAD

**Issue by axmand** [gpuweb/gpuweb#3003]
Is there a method similar to "gl.getActiveXXX" to help us analyze information such as "attributeInfo" in ShaderModule and return location/type so that GPUVertexBufferLayOut can be filled without revenue?

**Comment by Kangz** [comment]
No, there is no shader reflection at this time. See for example https://github.com/gpuweb/gpuweb/issues/2316 and https://github.com/gpuweb/gpuweb/issues/624

It is something to consider post-V1 though, but for v1 the standardization cost of shader reflection will delay the release of WebGPU too much.
